mathml/relations/css-styling/padding-border-margin/margin-002.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>margin</title>
+<link rel="help" href="https://mathml-refresh.github.io/mathml-core/#layout-algorithms">
+<meta name="assert" content="Verify that margin is taken into account.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/mathml/support/mathml-fragments.js"></script>
+<script src="helper.js"></script>
+<script>
+  var epsilon = 1;
+
+  setup({ explicit_done: true });
+  window.addEventListener("load", runTests);
+
+  function runTests() {
+
+    for (tag in MathMLFragments) {
+        if (!FragmentHelper.isValidChildOfMrow(tag))
+            continue;
+
+        var style = "margin-left: 30px; margin-right: 40px;  margin-top: 50px; margin-bottom: 60px;";
+
+        if (FragmentHelper.isEmpty(tag)) {
+            test(function() {
+                var s = compareSizeWithAndWithoutStyle(tag, style);
+                assert_approx_equals(s.width_delta, 30+40, epsilon, "left/right margin");
```

Comment:
We should decide on a consistent style for "num+num", i.e. I would do "num + num". I guess with different people writing tests and different tastes this can't be avoided though.

Reply:
Right, I generally write num + num